pkg/config/config_template.yaml

```
@@ -1061,7 +1061,9 @@ api_key:
   ## @env DD_APM_MAX_TPS - integer - optional - default: 10
   ## The target traces per second to sample. Sampling rates to apply are adjusted given
   ## the received traffic and communicated to tracers. This configures head base sampling.
-  ## Set to 0 to disable sampling.
+  ## As of 7.35.0 setting max_traces_per_second to 0 no longer disables sampling and will instead
+  ## send no traces to the intake. Sampling can not be disabled, to avoid rate limiting set this value
+  ## sufficiently high for your traffic pattern.
```

Minor language nit suggestion:
```suggestion
  ## As of 7.35.0 sampling cannot be disabled and setting 'max_traces_per_second' to 0 no longer 
  ## disables sampling but will instead send no traces to the intake. To avoid rate limiting, set this
  ## value sufficiently high for your traffic pattern.
```